tests/test_copyoffload_migration.py

```
@@ -2292,3 +2305,431 @@ def test_check_vms(
+
+    def test_create_storagemap_plan1(
+        self,
+        prepared_plan_1: dict[str, Any],
+        fixture_store: dict[str, Any],
+        ocp_admin_client: "DynamicClient",
+        source_provider: "BaseProvider",
+        destination_provider: "OCPProvider",
+        source_provider_inventory: "ForkliftInventory",
+        target_namespace: str,
+        source_provider_data: dict[str, Any],
+        copyoffload_storage_secret: Any,
```

Ruff (0.14.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `copyoffload_storage_secret`

(ANN401)